See a little issue in AppVeyor, though:

```
  Traceback (most recent call last):
    File "C:/projects/node-webrtc-7bnua/build/external/libwebrtc/download/src/build/vs_toolchain.py", line 577, in <module>
      sys.exit(main())
    File "C:/projects/node-webrtc-7bnua/build/external/libwebrtc/download/src/build/vs_toolchain.py", line 573, in main
      return commands[sys.argv[1]](*sys.argv[2:])
    File "C:/projects/node-webrtc-7bnua/build/external/libwebrtc/download/src/build/vs_toolchain.py", line 388, in CopyDlls
      _CopyDebugger(target_dir, target_cpu)
    File "C:/projects/node-webrtc-7bnua/build/external/libwebrtc/download/src/build/vs_toolchain.py", line 424, in _CopyDebugger
      (debug_file, full_path))
  Exception: api-ms-win-downlevel-kernel32-l2-1-0.dll not found in "C:\Program Files (x86)\Windows Kits\10\Debuggers\x64\api-ms-win-downlevel-kernel32-l2-1-0.dll"
  
  You must installWindows 10 SDK version 10.0.19041.0 including the "Debugging Tools for Windows" feature.
  ERROR at //build/toolchain/win/BUILD.gn:49:3: Script returned non-zero exit code.
    exec_script("../../vs_toolchain.py",
    ^----------
  Current dir: C:/projects/node-webrtc-7bnua/build/external/libwebrtc/build/Release/
```

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -9,7 +9,7 @@ environment:
       msvs_toolset: 15
       CHECK_REFLEXIVE: true
 
-os: Visual Studio 2017
+os: Visual Studio 2019
 
 install:
   - scripts\build-appveyor.bat

--- a/scripts/build-from-source.js
+++ b/scripts/build-from-source.js
@@ -13,8 +13,8 @@ if (process.env.DEBUG) {
 if (process.platform === 'win32') {
   args.push('-g');
   args.push(process.arch === 'x64'
-    ? '"Visual Studio 15 2017 Win64"'
-    : '"Visual Studio 15 2017"');
+    ? '"Visual Studio 16 2019 Win64"'
+    : '"Visual Studio 16 2019"');
 }
 
 function main() {

--- a/scripts/build-from-source.js
+++ b/scripts/build-from-source.js
@@ -12,9 +12,7 @@ if (process.env.DEBUG) {
 
 if (process.platform === 'win32') {
   args.push('-g');
-  args.push(process.arch === 'x64'
-    ? '"Visual Studio 16 2019 Win64"'
-    : '"Visual Studio 16 2019"');
+  args.push('"Visual Studio 16 2019"');
 }
 
 function main() {

